api/app/src/routes/medical/__tests__/fhir-r4/e2e/shared.ts

```diff
@@ -0,0 +1,14 @@
+import Axios from "axios";
+import { customAlphabet } from "nanoid";
+
+export const nanoid = customAlphabet("1234567890abcdef", 10);
+
+// TODO: How to make this dynamic? Create an acc for every test?
+const apiKey = "YThtMldReVhXdkF3eldXQ0lLR2ZVOjE1NDE0NTZmLTY3MzQtNDJhNS05ZWI3LWVmMGQ5OGIxMGQ0OQ";
```

Comment:
assuming this is a dummy non-prod key?

Reply:
yeah, this is a local API key, but I'll remove as it doesn't do anything on CI/CD - E2E tests don't run there.